### DISCUSSION TITLE
Tabular data

### DISCUSSION THREAD

**Discussion by maha-msft** [thongekchakrit/automated-data-analysis#1] [Ideas]
Hi @thongekchakrit 

Any workaround to display the result data in tabular format?

**Comment by thongekchakrit** [comment]
Hi maha-msft, for tabular data format we can include a https://mui.com/material-ui/react-table/. Take a look at an example by https://okld-gallery.streamlit.app/?p=elements, here, they labeled the tabular display as "data grid". I think they used MUI table, we can incorporate this into the tool.

The idea would be something along the line of:
1. Prompt engineer for GPT to give result in tabular format
2. Create a function that will transform the data as needed.
3. Add the function into the if-else statement similar like the rest of the code that will populate the table onto MUI Card
